pnpm-lock.yaml

```diff
@@ -1,8 +1,8 @@
 importers:
```

Comment:
I have no idea why there are so many lockfile changes. I've just executed `pnpm i` after fixing that workspace option - which had to be done, because it affects lockfile. Not sure why any versions here were changed though.